Create switch.ecoplug.markdown

adamkempenich opened the pull request:
Setup description for EcoPlug component in HomeAssistant.

**Description:**


**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15141
## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

adamkempenich committed:
Create switch.ecoplug.markdown

Setup description for EcoPlug component in HomeAssistant.

frenck commented on source/_components/switch.ecoplug.markdown:
```diff
@@ -0,0 +1,27 @@
+sharing: true
+footer: true
+ha_category: Switch
+ha_release: 0.71
```
`0.71` has been released already, maybe set this to the next major release, e.g., currently, `0.73`

adamkempenich replied:
Noted. Thank you! I'll make all of these changes after clearing the checklist for the device itself.

frenck commented on source/_components/switch.ecoplug.markdown:
````diff
@@ -0,0 +1,27 @@
+  - platform: ecoplug
+ ```
+
+Configuration variables:
````
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

frenck commented on source/_components/switch.ecoplug.markdown:
```diff
@@ -0,0 +1,27 @@
+ha_category: Switch
+ha_release: 0.71
+ha_iot_class: "Local Polling"
+---
```
Maybe add a logo?
That makes the documentation and the component a little more recognizable.

frenck reviewed (changes requested):
@adamkempenich Thank you for this!
I've reviewed your documentation PR and left a few comments. Could you please take a look? 👍

adamkempenich committed:
Add files via upload

adamkempenich committed:
Update switch.ecoplug.markdown

Should be good on both ends! Let me know if there are any more changes that you'd like to see.

frenck commented on source/_components/switch.ecoplug.markdown:
```diff
@@ -0,0 +1,27 @@
+ {% endconfiguration %}
+
+
+Configuration variables:
```
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

frenck commented on source/_components/switch.ecoplug.markdown:
```diff
@@ -0,0 +1,27 @@
+
+To enable this switch, you first have to set up your devices in the EcoPlug App, and add the following lines to your `configuration.yaml` file:
+
+{% configuration %}
```
The configuration tags are for creating a list of configuration variables/options.
This is an example, which should just use markdown code...

adamkempenich replied:
Thanks for bearing with me on this... updating now. I think I'm figuring it out, but as always, let me know if things are still off-target!

frenck reviewed (changes requested):
@adamkempenich Thank you for updating the PR.
Could you please take a look at the documentation link about configuration tags again?

In case of doubt, be sure to check out some example of other components.

adamkempenich committed:
Update switch.ecoplug.markdown

fabaff commented on source/_components/switch.ecoplug.markdown:
```diff
@@ -0,0 +1,68 @@
+
+
+
+---
```
Needs to be the file header.

frenck commented:
Closing this PR since the parent PR has been closed.
